LICENSE.txt

```diff
@@ -1,25 +1,18 @@
-Copyright (c) 2012-15, Members of the Raising the Floor Consortium.
+Copyright (c) 2012-17
+Members of the Raising the Floor Consortium
 All rights reserved.
 
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are met:
+This is a test version of GPII. You will install the software on your computer
+for testing, and uninstall it at the end of the test.
 
- * Redistributions of source code must retain the above copyright notice, this
-   list of conditions and the following disclaimer.
- * Redistributions in binary form must reproduce the above copyright notice,
-   this list of conditions and the following disclaimer in the documentation
-   and/or other materials provided with the distribution.
- * Neither the name of OCAD University nor the names of its contributors may
-   be used to endorse or promote products derived from this software without
-   specific prior written permission.
+Please do not share it or give it to anyone else, because this just a test
```

Comment:
But isn't this pull request targeted against master? I think I must be confused, which happens often. :)

Anyway, it seems this updated license is probably suitable for both master and the HST branch.